release.nix

```diff
@@ -0,0 +1,8 @@
+{ nixpkgs ? null, ... }@_args:
```

Comment:
What about importing/calling it from ./tests.nix so that we at least have a smoke test from there? (./tests.nix is also "built" when cargo test is run) Kind of imitating what hydra does. That way, I will not inadvertedly break it since I don't use hydra (yet?).